llvm_objects.py

```diff
@@ -162,7 +205,7 @@ def _get_instructions(self, l_block, w_module):
             prev = current_instr
             w_instrs.append(current_instr)
             l_instr = llwrap.LLVMGetNextInstruction(l_instr)
-        return w_instrs
+        return w_instrs[:]
```

Comment:
You should be able to put the copy in either position, caller or callee, no? Or RPython chokes?